plugins/inventory/icinga2.py

```diff
@@ -232,6 +251,9 @@ def _convert_inv(self, json_data):
                                         host_attrs['state'])
             self.inventory.set_variable(host_name, 'state_type',
                                         host_attrs['state_type'])
+            # Adds all attributes to a host variable 'icinga2'
+            self.inventory.set_variable(host_name, 'icinga2', host_attrs)
+            self._apply_constructable(host_name, self.inventory.get_host(host_name).get_vars())
```

Comment:
```suggestion
            construct_vars = dict(self.inventory.get_host(host_name).get_vars())
            construct_vars['icinga2'] = host_attrs
            self._apply_constructable(host_name, construct_vars)
```

Reply:
Thank you very much! I apply the changes you proposed to me. I decided that it would be better to change the name of the variable for more specifics.